src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -163,6 +163,11 @@ internal override CommonAnonymousTypeManager CommonAnonymousTypeManager
         /// </summary>
         internal bool FeatureStrictEnabled => Feature("strict") != null;
 
+        /// <summary>
+        /// True if we should enable nullable analysis in this compilation.
+        /// </summary>
+        internal bool EnableNullableAnalysis => Feature("run-nullable-analysis") is "true";
```

Comment:
>is "true" [](start = 81, length = 9)

`!= null` is the way we have been doing it (look at the previous property)

Reply:
Because `run-nullable-analysis` can be set to `false` or `true`, this won't work. I'll leave it as is.

---
In reply to: [276406471](https://github.com/dotnet/roslyn/pull/35047#discussion_r276406471) [](ancestors = 276406471)